src/server.rs

```diff
@@ -309,7 +314,9 @@ fn check_mutation_ticks(check: On<CheckChangeTicks>, mut clients: Query<&mut Cli
         check.present_tick()
     );
     for mut ticks in &mut clients {
-        ticks.check_mutation_ticks(*check);
+        for entity_ticks in ticks.entities.values_mut() {
+            entity_ticks.system_tick.check_tick(*check);
+        }
```

Comment:
This logic should be inside `ClientTicks`.

Reply:
It's a tiny loop and since we already expose the access to `tick.entities`, I think it reads better this way.

Reply:
Ah, I see related suggestions below that might let us hide it. I'll move the logic.